java/adapter/jdbc/src/test/java/org/apache/arrow/adapter/jdbc/AbstractJdbcToArrowTest.java

```diff
@@ -91,15 +89,15 @@ protected static Table getTable(String ymlFilePath, @SuppressWarnings("rawtypes"
             clss.getClassLoader().getResourceAsStream(ymlFilePath), Table.class);
   }
 
-
   /**
    * This method creates Connection object and DB table and also populate data into table for test.
    *
    * @throws SQLException on error
    * @throws ClassNotFoundException on error
    */
-  @Before
-  public void setUp() throws SQLException, ClassNotFoundException {
+  protected void initializeDatabase(Table table) throws SQLException, ClassNotFoundException {
```

Comment:
@vibhatha I'll look into it more, but if you have any suggestions for a better structure, I would really appreciate your advice.

cc @lidavidm

```mermaid
classDiagram
    direction LR
    class AbstractJdbcToArrowTest {
    }

    class JdbcToArrowCharSetTest {
    }

    class JdbcToArrowDataTypesTest {
    }

    class JdbcToArrowMapDataTypeTest {
    }

    class JdbcToArrowNullTest {
    }

    class JdbcToArrowOptionalColumnsTest {
    }

    class JdbcToArrowTest {
    }

    class JdbcToArrowTimeZoneTest {
    }

    class JdbcToArrowVectorIteratorTest {
    }

    AbstractJdbcToArrowTest <|-- JdbcToArrowCharSetTest
    AbstractJdbcToArrowTest <|-- JdbcToArrowDataTypesTest
    AbstractJdbcToArrowTest <|-- JdbcToArrowMapDataTypeTest
    AbstractJdbcToArrowTest <|-- JdbcToArrowNullTest
    AbstractJdbcToArrowTest <|-- JdbcToArrowOptionalColumnsTest
    AbstractJdbcToArrowTest <|-- JdbcToArrowTest
    AbstractJdbcToArrowTest <|-- JdbcToArrowTimeZoneTest
    JdbcToArrowTest <|-- JdbcToArrowVectorIteratorTest
```

Reply:
@llama90 I will review this tomorrow. Thanks.

Reply:
I guess what you are concerned about is making the database connection for each test case with `@BeforeEach` setup method?